src/uploaders.rs

```diff
@@ -83,30 +83,29 @@ impl Uploader {
 
     /// Uploads a file using the configured uploader (either `S3`, `Local`).
     ///
-    /// It returns a a tuple containing the path of the uploaded file
-    /// and its checksum.
-    pub fn upload(
+    /// It returns the path of the uploaded file.
```

Comment:
As far as I can tell, the return value is never used, so we could just as well get rid of it.